docs/cookbook/creating_custom_notifications_for_flex_workflow.md

```diff
@@ -1,19 +1,26 @@
 # Creating custom notifications for Flex Workflow
 
-To create a custom notification you have to provide two plugins in the `flex-workflow` bundle, one for the `notificationsPopupView` and second for the `notificationIndicatorView`.
+!!! note
 
-We will start from creating a plugin for `notificationIndicatorView` which is responsible for displaying a text in the notification bar (this green stripe in the bottom when you get new notification)
+    This recipe assumes you have a back end for notifications and want to display them
```

Comment:
For search purpose, the usage of _back end_ is not a good thing. I suggest _backend_ or at least _back-end_.

Reply:
I've done the same change in previous audits (e.g. here: https://github.com/ezsystems/developer-documentation/pull/74/files#diff-f7003b59463affa0e04aa6b72784e75dR380)
I don't think there's a huge difference between the two versions (although "back end" is accepted by more dictionaries), but we should stick to one version throughout.